core/network/types/collator_messages.hpp

```diff
@@ -182,40 +182,251 @@ namespace kagome::network {
     Signature signature;
   };
 
-  struct Seconded {
+  struct SignedStatement {
     SCALE_TIE(2);
 
     primitives::BlockHash relay_parent;  /// relay parent hash
     Statement statement;                 /// statement of seconded candidate
   };
 
+  /// Seconded is an alias for signed statement.
+  using Seconded = SignedStatement;
+
+  struct BitfieldData {
+    SCALE_TIE(3);
+
+    std::vector<bool> bitfield;   /// The availability bitfield
+    ValidatorIndex validator_ix;  /// Validator index in the authority set.
+    Signature signature;          /// Signature of the validator.
+  };
```

Comment:
Better move non message types for reuse to "parachain/types.hpp" into "namespace kagome::parachain"?
```suggestion
  struct SignedBitfield {
    SCALE_TIE(3);

    std::vector<bool> bitfield;   /// The availability bitfield
    ValidatorIndex validator_ix;  /// Validator index in the authority set.
    Signature signature;          /// Signature of the validator.
  };
```